package.json

```diff
@@ -39,13 +39,14 @@
   "sideEffects": false,
   "type": "module",
   "main": "index.js",
-  "types": "types/index.d.ts",
+  "types": "index.d.ts",
   "files": [
-    "types/index.d.ts",
+    "lib/",
     "index.d.ts",
     "index.js"
   ],
   "dependencies": {
+    "@types/extend": "^3.0.0",
```

Comment:
These types aren’t used in the unified type definitions. This makes it a dev dependency.

```suggestion
```